dist/gmusic.js

```diff
@@ -211,13 +211,16 @@ proto.playback = {
   toggleShuffle: function () { this.playback._shuffleEl.click(); },
 
   getRepeat: function () {
-    var title = this.playback._repeatEl.getAttribute('title').toLowerCase();
-    if (title.indexOf('repeat off') !== -1) {
-      return GMusic.Playback.NO_REPEAT;
-    } else if (title.indexOf('repeating all') !== -1) {
+    // States possible:
+    // no-repeat: No class 'active', icon: av:repeat
+    // repeat: Has class 'active', icon: av:repeat
+    // repeat-one: Has class 'active', icon: av:repeat-one
+    if (this.playback._repeatEl.__data__.icon === 'av:repeat-one') {
```

Comment:
Good documentation but here's some feedback to prevent confusion between the HTML attribute `icon` and JS one:

- Indent related subcomments
- Keep our mindset in JS logic by providing code sames instead of has class/no class
- Use `el.__data__.icon` with an equality setter
- Match order with following if/elses

```js
// Repeat element states:
//   SINGLE_REPEAT: el.classList.has('active'); el.__data__.icon === 'av:repeat-one'
//   LIST_REPEAT: el.classList.has('active'); el.__data__.icon === 'av:repeat'
//   NO_REPEAT: !el.classList.has('active'); el.__data__.icon === 'av:repeat'
```